userHome.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @Author : Mfsi_Annapurnaa
+ * @purpose : Display after login
+ */
+
+require_once('config/session.php');
+
+$objSes = new session();
+$objSes->start();
+$resultSes = $objSes->checkSession();
+
+if (!$resultSes)
+{
+    header('Location:login.php');
+}
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="">
+        <meta name="author" content="">
+        <title>Get Employed</title>
+        <!-- Bootstrap Core CSS -->
+        <link href="css/bootstrap.min.css" rel="stylesheet">
+        <!-- Custom CSS -->
+        <link href="css/styles.css" rel="stylesheet">
+    </head>
+    <body>
+        <?php
+        include('template/header.php');
```

Comment:
Many places i see include and not require_once, so is there any specific reason for using include?